disable-mqtt.js

```diff
@@ -0,0 +1,14 @@
+var fs = require('fs');
+// NOTE(bajtos) By default, appmetrics is configured to connect to a local
+// MQTT broker. The only reliable way how to disable this behaviour is to edit
+// appmetrics.properties in module's installation directory.
+try {
+  var propFile = require.resolve('appmetrics/appmetrics.properties');
+  var content = fs.readFileSync(propFile, 'utf-8');
+  content = content.replace(
+    /com.ibm.diagnostics.healthcenter.mqtt\s*=\s*on/,
+    'com.ibm.diagnostics.healthcenter.mqtt=off');
+  fs.writeFileSync(propFile, content);
+} catch (err) {
+  console.warn('Cannot modify appmetrics.properties:', err.stack);
```

Comment:
can you remove the stack trace, and just put in err.message? this will happen all the time with a normal node installation, in which sudo is used to install, because scripts run without the ability to modify the fs (sudo perms are dropped before the script is run). err stacks trigger support calls, but a warning like above will be less problematic.